src/main/java/invtweaks/InvTweaksConfigManager.java

```diff
@@ -215,8 +217,9 @@ private boolean loadConfig() {
         }
     }
 
-    private boolean extractFile(ResourceLocation resource, File destination) {
-        try(InputStream input = mc.getResourceManager().getResource(resource).getInputStream()) {
+    private boolean extractFile(String resourcePath, File destination) {
+        ClassLoader classloader = Thread.currentThread().getContextClassLoader();
+        try (InputStream input = classloader.getResourceAsStream(resourcePath)) {
```

Comment:
Everything in this whole file (and things related to it) should all be unnecessary -- it's using the resource system to (theoretically) allow overriding the default, even if the configurations get reset.

(There's *probably* issues with doing that in practice due to how early the initial load usually is, but that's still no reason to randomly change it)